### DISCUSSION TITLE
Working - and not Working - Hardware Combinations

### DISCUSSION THREAD

**Discussion by ruebyi** [outdoorbits/little-backup-box#140] [Show and tell]
Hi, 

I promised to test out some Hardware and give Feedback:

Samsung T5 2TB -> directly at RPi 4 4GB NOT WORKING
Samsung T5 2TB -> Orico USB Hub (Link in Wiki) - WORKING

SanDisk Extreme 128 micro SD -> Ugreen SD / microSD Card Reader to Orico USB Hub -> WORKING
Lexar UHS II x1667 128GB -> Ugreen SD / microSD Card Reader to Orico USB Hub -> WORKING
Transcend Premium 16GB micro SD -> Ugreen SD / microSD Card Reader to Orico USB Hub -> WORKING
Transcend UHS II R285 (golden one) -> Ugreen SD / microSD Card Reader to Orico USB Hub -> WORKING

I will run some more tests with my Olympus E-M1 II, Olympus E-M1 III and E-M5 III

I'm using a dual USBC AC-Adapter with 2 USB-C Cables reaching out to the Pi4 and the Orico USB hub

**Comment by ruebyi** [comment]
E-M1 II , E-M1 III , E-M5 III, DJI Mini 2, GoPro Hero 8 and GoPro Hero 5 work via Cardreader

Zoom H1n, Zoom H1 and Zoom H4 work via Cardreader
Olympus LP4 works via Cardreader

**Comment by outdoorbits** [comment]
Great job, thank you very much! I just uploaded the changes at 
https://github.com/outdoorbits/little-backup-box/wiki/02.-Hardware. Hope 
everything is correct.

**Comment by ruebyi** [comment]
The least I can do!
If you could tell me how to help you with the translation I would make some suggestions for German.

**Comment by outdoorbits** [comment]
You are very welcome! It's a bit funny, I'm also a German native 
speaker. ;-) But to save time, indeed I did all the translations from 
English by Google translater and only a few of them I changed. I would 
be very grateful for a more natural tanslation but actually I don't find 
the time. Technically it's very easy.

There are at least two possible ways:

1. Create a fork of Little Backup Box, edit the file in your fork 
directly at github and make a pull request - everything online in 
github. Or just tell me when I can merge your work.
2. Download the file, edit it and send it by mail, I will merge it.

**Comment by munecito** [comment]
Samsung T5 1TB connected to Pi USB3 port. Raw steel USB 3 card reader (old model) connected to Pi USB3 port, Sony SD Tough 128Gb connected to card reader. Working.

Samsung Evo SSD SATA in a USB 3 sata to USB enclosure connected to pi USB3 port. Raw steel USB 3 card reader (old model) connected to Pi USB3 port, Sony SD Tough 128Gb connected to card reader. NOT Working.

**Comment by munecito** [comment]
Raspberry pi 4 B 8GB with Raspberry Pi OS 64bit full (desktop) hdmi to TV WORKING. Can load LBB screen by calling https://MY.IP.ADDRESS directly on the Pi's browser.

It would be great if a graphic interface was added without having to do it by the browser.

**Comment by munecito** [comment]
Pi-Top 4 Seems to be working fine if running Raspberry Pi OS 32bit. 
Raspberry pi 4 B 8Gb
Samsung pro endurance micro sd 32Gb
Samsung T5 1tb
USB to Sata (Samsung Sata SSD) 960gb

The Pi-Top 4 DIY kit is expensive but comes with cooling, case, battery, etc. Seems like a good solution, but more expensive than making your own. It seems to be able to feed enough power to the Sata to Usb drive.

I am yet to try to use the OLED screen of the Pi-Top to display the information of the process. 

I could not get it to work with the Pi-Top OS (Based on raspberry pi OS) for some reason it won't let me do the first Wifi configuration. I will keep trying.

**Reply by edenraf** [reply]
Have you had any luck getting the information to be displayed on an OLED screen?

**Reply by munecito** [reply]
I haven't looked into it

**Reply by edenraf** [reply]
Looks lke its built in now with an I2C OLED screen as shown in the Wiki, which I missed earlier

**Reply by munecito** [reply]
I didn't build one but I have one running in a pi-top 4. But I haven't checked if the OLED screen in it can work with the lbb

**Comment by brentener** [comment]
Hello !
I have no luck installing little backup box on my raspberry 4. I get an error message at the end of the installation that a specific url is unavailable ! Could it be that the installation link is outdted ?

**Comment by outdoorbits** [comment]
Did you install via wlan and did you select to install comitup?

**Reply by brentener** [reply]
wlan, i am just running the installation again ! i will send you the error !

**Comment by brentener** [comment]
It seems to work this time ! The oled display says: rebooting.... and the terminal is stuck. I will keep you updated !!!!

**Comment by brentener** [comment]
I have a working Little Backup Box now. I am connected !!!!!

**Comment by outdoorbits** [comment]
Hi,

most probably as a user I think, I wouldn`t have read that, but does the 
following specify the problem and the solution? If yes, I think it would 
an improvement to print this on installation-screen. Can you confirm 
that?

Don't worry: Please note this in case you installed via wlan/ssh and 
decided to install comitup. If your installation-screen scrolls down and 
seems to freeze at something like _Setting up python3-networkmanager 
(2.2-1) ..._, this doesn't mean anything went wrong. Comitup is 
installed at the end of the process and when it's comming up, it changes 
your wlan-network. Your ssh-session is broken and you will not get any 
more information. In the background the installation finishes and a few 
seconds later the pi reboots. What you can't see anymore is just this:

********************************************************************************************
*** How to proceed:
*** Assuming your Little Backup Box gets the same IP address again after 
the reboot,
*** you can then reach the web UI as follows:
***
*** https://10.41.0.1 (secure, certificate cannot be verified 
automatically, please confirm it)
*** http://10.41.0.1:8000 (insecure)
***
*** Please use the settings of the web UI to optimally adapt the Little 
Backup Box for you.
***
*** If you have further questions, please check the wiki first:
*** https://github.com/outdoorbits/little-backup-box/wiki
***
*** We are always happy to receive your feedback
********************************************************************************************
... using comitup, you now have to connect your computer to the wlan 
_little-backup-box-nnnn_ to access the new IP 10.41.0.1.

**Comment by wrightmac** [comment]
Hello! Wow what a great setup. This fits the bill perfect for my photographer wife. She recently lost an SD card and she doesn’t do that. 
I have put together a proof of concept over the last couple of days. A RPi 3B+, SanDisk 128GB thumb drive, and a SD card reader. 
and it doesn’t get much easier. Turn it on, put in the card, wait for it to turn off. 😱and the card is backed up. I have been beating on it pretty good the past couple of evenings and quite happy. 
This is now going towards a full blown project. 
has anyone tried any other SBC/OS combinations? With any Pi just not reasonably available I’m curious if there are other options. 
Keep up the great work! And I’ll keep exercising it.

**Reply by munecito** [reply]
> Hello! Wow what a great setup. This fits the bill perfect for my photographer wife. She recently lost an SD card and she doesn’t do that. I have put together a proof of concept over the last couple of days. A RPi 3B+, SanDisk 128GB thumb drive, and a SD card reader. and it doesn’t get much easier. Turn it on, put in the card, wait for it to turn off. 😱and the card is backed up. I have been beating on it pretty good the past couple of evenings and quite happy. This is now going towards a full blown project. has anyone tried any other SBC/OS combinations? With any Pi just not reasonably available I’m curious if there are other options. Keep up the great work! And I’ll keep exercising it.

You could try using an orange pi. They seem to have an ok support and several OS options.
